### PR TITLE
Preprocess available time zones on Windows #2673

### PR DESCRIPTION
**Pull request by joachimmetz**
Preprocess available time zones on Windows #2673

### CODE REVIEW

File: plaso/cli/pinfo_tool.py [hunk 1]
```diff
@@ -494,7 +519,7 @@ def _PrintSessionsDetails(self, storage_reader):
     Args:
       storage_reader (BaseStore): storage.
     """
-    for session_number, session in enumerate(storage_reader.GetSessions()):
+    for _, session in enumerate(storage_reader.GetSessions()):
```
Comment (Onager) [inline plaso/cli/pinfo_tool.py]: Remove the enumerate here, it's not needed any more

File: plaso/containers/artifacts.py [hunk 2]
```diff
@@ -276,7 +295,7 @@ class UserAccountArtifact(ArtifactAttributeContainer):
   def __init__(
       self, full_name=None, group_identifier=None, identifier=None,
       path_separator='/', user_directory=None, username=None):
-    """Initializes an user artifact.
+    """Initializes an user account artifact.
```
Comment (Onager) [inline plaso/containers/artifacts.py]: "...a user account..."

File: plaso/cli/pinfo_tool.py [hunk 3]
```diff
@@ -377,7 +378,8 @@ def _PrintEventLabelsCounter(
     Args:
       event_labels_counter (collections.Counter): number of event tags per
           label.
-      session_identifier (Optional[str]): session identifier.
+      session_identifier (Optional[str]): session identifier, formatted as
+          an UUID.
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply plaso/cli/pinfo_tool.py]: ack

File: plaso/cli/pinfo_tool.py [hunk 4]
```diff
@@ -433,24 +436,30 @@ def _PrintParsersCounter(self, parsers_counter, session_identifier=None):
...
     knowledge_base_object = knowledge_base.KnowledgeBase()
 
     storage_reader.ReadPreprocessingInformation(knowledge_base_object)
 
-    # TODO: replace session_number by session_identifier.
+    # The knowledge base requires the session identifier to be formatted in
+    # hexadecimal representation.
     system_configuration = knowledge_base_object.GetSystemConfigurationArtifact(
-        session_identifier=session_number)
+        session_identifier=session_identifier.replace('-', ''))
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply plaso/cli/pinfo_tool.py]: good catch

File: plaso/cli/pinfo_tool.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/cli/pinfo_tool.py]: ack

File: plaso/containers/artifacts.py (hunk 2 above)
Comment (joachimmetz) [reply plaso/containers/artifacts.py]: ack

**Pull request merged**